cherrypicks_test.go

```diff
@@ -21,12 +20,21 @@ import (
 	"github.com/mendersoftware/integration-test-runner/logger"
 )
 
-func TestSuggestCherryPicks(t *testing.T) {
+const versionsResponse = `{
+	"lts": ["3.0", "3.3"],
```

Comment:
Suggestion (nit pick): to make the messages closer to the production ones, order these from most recent to older
```suggestion
	"lts": ["3.3", "3.0"],
```

Reply:
Ah, so my guess was incorrect 🤣